backend/src/apps/api/v1/ideas.py

```diff
@@ -0,0 +1,63 @@
+from django.urls import path, include
+from drf_spectacular.utils import extend_schema, extend_schema_view
+from rest_framework import status
+
+from apps.ideas.models import Idea
+from apps.core.views.base import BaseOrganizationModelView, BaseOrganizationDetailView
+from apps.api.serializers.ideas import IdeaSerializer
+
+@extend_schema(tags=["Идеи"])
+@extend_schema_view(
+    post=extend_schema(
+        summary="Создание идеи",
+        request=IdeaSerializer,
+        responses={status.HTTP_201_CREATED: IdeaSerializer},
+    ),
+    get=extend_schema(
+        summary="Получение списка всех идей",
+        responses={status.HTTP_200_OK: IdeaSerializer(many=True)},
+    ),
+)
+class OrganizationIdeasAPI(BaseOrganizationModelView):
+    queryset = Idea.objects.all()
+    serializer_class = IdeaSerializer
+
+    def perform_create(self, serializer):
+        serializer.save(author=self.request.user)
+
+@extend_schema(tags=["Идеи"])
+@extend_schema_view(
+    patch=extend_schema(
+        summary="Обновление идеи",
+        request=IdeaSerializer,
+        responses={status.HTTP_200_OK: IdeaSerializer},
+    ),
+    get=extend_schema(
+        summary="Получение информации об идее",
+        responses={status.HTTP_200_OK: IdeaSerializer},
+    ),
+    delete=extend_schema(
+        summary="Удаление идеи",
+        responses={status.HTTP_204_NO_CONTENT: None},
+    ),
+)
+class IdeasAPI(BaseOrganizationDetailView):
```

Comment:
В данном случае нужен RetrieveUpdateDestroyAPIView

```suggestion
class IdeasDetailAPI(RetrieveUpdateDestroyAPIView):
```